extensions/jaeger/runtime/src/main/java/io/quarkus/jaeger/runtime/QuarkusJaegerTracer.java

```diff
@@ -34,7 +34,7 @@ public QuarkusJaegerTracer() {
 
     @Override
     public String toString() {
-        return "Jaeger Tracer";
+        return tracer().toString();
```

Comment:
Can we be sure that `tracer()` will never be `null`?

Reply:
It should not be `null`. If it is `null` a new instance is created.

Before running the creation there is a check to verify the configuration is correct.